keyboards/handwired/illera16/rules.mk

```diff
@@ -0,0 +1,5 @@
+# This file intentionally left blank
+
+RGBLIGHT_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
`ENCODER_MAP_ENABLE` cannot be enabled at the keyboard level, the rest should be in info.json
```suggestion
```